subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/testsuites/dependencies/TestSuitesGroovyDSLDependenciesIntegrationTest.groovy

```diff
@@ -749,48 +749,6 @@ class TestSuitesGroovyDSLDependenciesIntegrationTest extends AbstractIntegration
         }
         """
 
-        file('src/main/org/sample/Person.java') << """
-            package org.sample;
-
-            public class Person {
-                private String name;
-                private int age;
-
-                public String getName() {
-                    return name;
-                }
-
-                public void setName(String name) {
-                    this.name = name;
-                }
-
-                public int getAge() {
-                    return age;
-                }
-
-                public void setAge(int age) {
-                    this.age = age;
-                }
-            }
-        """
-
-        file('src/test/org/samplePersonTest.java') << """
-            package org.sample;
-
-            import org.apache.commons.beanutils.PropertyUtils;
-
-            public class PersonTest {
-                @Test
-                public void testPerson() {
-                    Object person = new Person();
-                    PropertyUtils.setSimpleProperty(person, "name", "Bart Simpson");
-                    PropertyUtils.setSimpleProperty(person, "age", 38);
-                    assertEquals("Bart Simpson", person.getName());
-                    assertEquals(38, person.getAge());
-                }
-            }
-        """
-
```

Comment:
Why was this removed?

Reply:
The code wasn't actually used, as it wasn't in a `java` sub-folder. Tom confirmed that it didn't matter for the purposes of this test.